manager/integration/tests/test_node.py

```diff
@@ -4,21 +4,32 @@
 import subprocess
 
 from common import client  # NOQA
-from common import Gi, SIZE, CONDITION_STATUS_FALSE, CONDITION_STATUS_TRUE
+from common import Gi, SIZE, CONDITION_STATUS_FALSE, \
+    CONDITION_STATUS_TRUE, DEFAULT_DISK_PATH
 from common import get_self_host_id
 from common import SETTING_STORAGE_OVER_PROVISIONING_PERCENTAGE, \
     SETTING_STORAGE_MINIMAL_AVAILABLE_PERCENTAGE
 from common import get_volume_endpoint
+from common import get_update_disks
+from common import wait_for_disk_status, wait_for_disk_update, \
+    reset_disks_for_all_nodes
 
 SMALL_DISK_SIZE = (1 * 1024 * 1024)
-DEFAULT_DISK_PATH = '/var/lib/rancher/longhorn/'
 TEST_FILE = 'test'
 
 DISK_STATE = "state"
 DISK_STATE_SCHEDULABLE = "schedulable"
 DISK_STATE_UNSCHEDULABLE = "unschedulable"
 
 
+def setup_function(function):
```

Comment:
Should not be necessary since the finalizer for the client/clients already included it.